google/cloud/security/common/data_access/dao.py

```diff
@@ -42,6 +38,7 @@
     'org_iam_policies': create_tables.CREATE_ORG_IAM_POLICIES_TABLE,
     'projects': create_tables.CREATE_PROJECT_TABLE,
     'project_iam_policies': create_tables.CREATE_PROJECT_IAM_POLICIES_TABLE,
+    # pylint: disable=line-too-long
```

Comment:
Can you add TODOs on the common pylint errors that you don't correct? They should be fixed in a followup PR.